.buildkite/scripts/pull-request/pipeline.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -euo pipefail
+
+npm install -g bun
```

Comment:
Is it worth pre-installing this on our agents so we don't have to do this at execution time?

Reply:
It's the default k8s agent that runs `pipeline upload`, which they're trying to keep small. We could create our own container image for it, though, if we start seeing download failures or similar. Speed-wise: bun is a small, single binary, so it's fast.

Reply:
Got it. Makes sense.